test/test_unit_api.py

```diff
@@ -43,7 +43,7 @@
 }
 
 
-MOCK_HOSTFACTORY_OBJECT = CreateTokenData(host_factory="some_host_factory")
+MOCK_HOSTFACTORY_OBJECT = CreateTokenData(host_factory="some_host_factory", cidr="1.2.3.4,0.0.0.0")
```

Comment:
Please add the following tests:
1.2.3
1.2.3/16
1.2.3.4,1.2.3 (to verify if it put the right one only , or ignore entire list) 
without cdr (the one you removed) verify the default is working

Reply:
great thanks!